h/schemas/api/group.py

```diff
@@ -2,12 +2,13 @@
 
 from __future__ import unicode_literals
 
-from h.schemas.base import JSONSchema
+from h.schemas.base import JSONSchema, ValidationError
```

Comment:
Overall note to self: I should be using `i18n` on the exception messages throughout. Noted.